docs/berkeley-upgrade/migrating-archive-database-to-berkeley.mdx

```diff
@@ -13,182 +13,313 @@ keywords:
+an interface which script provides. Please refer to below steps:
 
 #### Stage 1: Initial migration
 
-This stage is the very first one, which requires the initial `berkeley` schema. It is a foundation for the next migration stage because it populates the migrated database and creates an initial checkpoint for further incremental migration.
+This stage is the very first one, which requires the initial `berkeley` schema only. It is a foundation for the next migration stage because it populates the migrated database and creates an initial checkpoint for further incremental migration.
 
 - Inputs
    - Unmigrated mainnet database
    - Mainnet genesis ledger 
-   - Empty `berkeley` schema 
+   - Empty target `berkeley` database (empty means with schema created but without any content)
 
 - Outputs
-   - Migrated `mainnet` database to the `berkeley` database from genesis up to the last canonical block
+   - Migrated `mainnet` database to `berkeley` format from genesis up to the last canonical block in orginal database
    - Replayer checkpoint, which can be used for incremental migration
 
 ##### Phase 1: Berkeley migration app run
 
 ```
-mina-berkeley-migration --batch-size 1000 --config-file genesis_ledgers/mainnet.json
---mainnet-archive-uri {mainnet_connection_string} --migrated-archive-uri {migrated_connection_string} --mainnet-blocks-bucket 
-{bucket name} --network "mainnet"
+mina-berkeley-migration \
+   --batch-size 1000 \
+   --config-file genesis_ledgers/mainnet.json \
+   --mainnet-archive-uri {mainnet_connection_string} \
+   --migrated-archive-uri {migrated_connection_string} \
+   --blocks-bucket {bucket name} \
+   --network "mainnet"
 ```
 where: 
-`--network` - precomputed blocks prefix 
+`--network` - network name when determining precomputed blocks. There is an assumption that precomputed blocks are named with format: `{network}-{height}-{state_hash}.json`;
+
 
 
 ##### Phase 2: Replayer in migration mode run
 
 Replayer config must contain the Mainnet ledger as the starting point. So first, you must prepare the replayer config file:
 
 ```
- jq '.ledger.accounts' mainnet.json | jq  '{genesis_ledger: {accounts: .}}' > replayer_input_config.json"
+ jq '.ledger.accounts' genesis_ledger.json | jq  '{genesis_ledger: {accounts: .}}' > replayer_input_config.json
 ```
 
+where:
+   genesis_ledger.json is genesis file from which deamon bootstrap on particular network
+
 Then:
 ```
- mina-replayer --migration-mode --archive-uri {migrated_connection_string}  --input-file replayer_input_config.json --checkpoint-interval 10000
+ mina-replayer \
+   --migration-mode \
+   --archive-uri {migrated_connection_string} \
+   --input-file replayer_input_config.json \
+   --checkpoint-interval 10000
+```
+
+where:
+ - `archive-uri` - connection string to archive datbase 
+ - `output-file` - file which will hold ledger with auxiliary information like global slot and blockchain height which will be dumped on last block
```

Comment:
Why do we need `output-file` here. I don't think we ever use `output-file` flag in replayer.